pkg/machine/config.go

```diff
@@ -237,6 +238,15 @@ func ConfDirPrefix() (string, error) {
 	return confDir, nil
 }
 
+// GuardedRemoveAll functions much like os.RemoveAll but
+// will not delete certain catastrophic paths.
+func GuardedRemoveAll(path string) error {
+	if path == "" || path == "/" {
+		return fmt.Errorf("refusing to recusively delete `%s`", path)
```

Comment:
```suggestion
		return fmt.Errorf("refusing to recusively delete %q", path)
```

`%q` will do the quoting for us.